content/documentation/pages/1-installation/1-local/2-docker-customize.md

```diff
@@ -0,0 +1,200 @@
+              - '8181'
+            ports:
+              - '8181:8181'
+            environment:
+              - metrics.prometheus.target.refresh.cron=0/20 * * * * *
+              - metrics.prometheus.target.discovery.url=http://localhost:9393/runtime/apps
+              - metrics.prometheus.target.file.path=/tmp/targets.json
+            depends_on:
+              - dataflow-server
+
+    With the following:
+
+          influxdb:
+            image: influxdb:1.7.4
+            container_name: 'influxdb'
+            ports:
+              - '8086:8086'
+
+2.  In the `dataflow-server` services configuration block, replace the
+    following `environment` entries:
+
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.prometheus.enabled=true
+              - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.streamapp.security.enabled=false
+              - spring.cloud.dataflow.applicationProperties.stream.management.endpoints.web.exposure.include=prometheus,info,health
+
+    With the following:
+
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.enabled=true
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.db=myinfluxdb
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.uri=http://influxdb:8086
+
+3.  Modify the `grafana` service definition `image` attribute to replace
+    `spring-cloud-dataflow-grafana-prometheus` with
+    `spring-cloud-dataflow-grafana-influxdb`.
+
+## Accessing the host file system
+
+If you are developing custom applications on your local machine, you will need to register them with Data Flow.
+Data Flow is running inside of a Docker container, so the Docker container will will need access to your local file system in order to resolve the registration reference.
+
+You can enable local disk access from Docker container running the Data Flow Server by following these steps.
+
+1.  Mount the source host folders to the `dataflow-server` container.
+
+    For example, if the `my-app.jar` is in the `/thing1/thing2/apps`
+    folder on your host machine, add the following `volumes` block to
+    the `dataflow-server` service definition:
+
+          dataflow-server:
+            image: springcloud/spring-cloud-dataflow-server:${DATAFLOW_VERSION}
+            container_name: dataflow-server
+            ports:
+              - "9393:9393"
+            environment:
+              - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.brokers=kafka:9092
+              - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.zkNodes=zookeeper:2181
+            volumes:
+              - /foo/bar/apps:/root/apps
+
+This configuration provides access to the `my-app.jar` (and the other files in the folder) from within container’s `/root/apps/` folder.
```

Comment:
Instead of `my-app.jar`, would we rather say, "This configuration provides access to the directory(`/foo/bar`) containing your boot app jars, from within the container's `/root/apps` folder."

Reply:
ok